pkg/trace/api/api_test.go

```diff
@@ -586,6 +588,253 @@ func TestHandleStats(t *testing.T) {
+	defer func(old string) { info.GitCommit = old }(info.GitCommit)
+	defer func(old string) { info.BuildDate = old }(info.BuildDate)
+	ff, ok := os.LookupEnv("DD_APM_FEATURES")
+	if !ok {
+		defer os.Unsetenv("DD_APM_FEATURES")
+	} else {
+		defer func(old string) { os.Setenv("DD_APM_FEATURES", old) }(ff)
+	}
+	os.Setenv("DD_APM_FEATURES", "feature_flag")
+	info.Version = "0.99.0"
+	info.GitCommit = "fab047e10"
+	info.BuildDate = "2020-12-04 15:57:06.74187 +0200 EET m=+0.029001792"
+	h := rcv.makeInfoHandler()
+	rec := httptest.NewRecorder()
+	req := httptest.NewRequest("GET", "/info", nil)
+	h.ServeHTTP(rec, req)
+	if rec.Body.String() != `{
+	"Version": "0.99.0",
+	"GitCommit": "fab047e10",
+	"BuildDate": "2020-12-04 15:57:06.74187 +0200 EET m=+0.029001792",
+	"Endpoints": [
+		"/v0.3/traces",
+		"/v0.3/services",
+		"/v0.4/traces",
+		"/v0.4/services",
+		"/v0.5/traces",
+		"/v0.5/stats",
+		"/profiling/v1/input"
+	],
+	"FeatureFlags": [
+		"feature_flag"
+	],
+	"Config": {
+		"Enabled": true,
+		"Hostname": "test.host.name",
+		"DefaultEnv": "prod",
+		"ConfigPath": "/path/to/config",
+		"Endpoints": [
+			{
+				"Host": "https://target-intake.datadoghq.com",
+				"NoProxy": true
+			}
+		],
+		"BucketInterval": 1000000000,
+		"ExtraAggregators": [
+			"agg:val"
+		],
+		"ExtraSampleRate": 2.4,
+		"TargetTPS": 11,
+		"MaxEPS": 12,
+		"ReceiverHost": "localhost",
+		"ReceiverPort": 8111,
+		"ReceiverSocket": "/sock/path",
+		"ConnectionLimit": 12,
+		"ReceiverTimeout": 100,
+		"MaxRequestBytes": 123,
+		"StatsWriter": {
+			"ConnectionLimit": 20,
+			"QueueSize": 12,
+			"FlushPeriodSeconds": 14.4
+		},
+		"TraceWriter": {
+			"ConnectionLimit": 21,
+			"QueueSize": 13,
+			"FlushPeriodSeconds": 15.4
+		},
+		"ConnectionResetInterval": 0,
+		"StatsdHost": "stastd.localhost",
+		"StatsdPort": 123,
+		"LogLevel": "WARN",
+		"LogFilePath": "/path/to/logfile",
+		"LogThrottling": false,
+		"MaxMemory": 1000000,
+		"MaxCPU": 12345,
+		"WatchdogInterval": 60000000000,
+		"SkipSSLValidation": false,
+		"Ignore": {
+			"K": [
+				"1",
+				"2"
+			]
+		},
+		"ReplaceTags": [
+			{
+				"Name": "a",
+				"Pattern": "*",
+				"Re": null,
+				"Repl": "b"
+			}
+		],
+		"AnalyzedRateByServiceLegacy": {
+			"X": 1.2
+		},
+		"AnalyzedSpansByService": {
+			"X": {
+				"Y": 2.4
+			}
+		},
+		"DDAgentBin": "/path/to/core/agent",
+		"Obfuscation": {
+			"ES": {
+				"Enabled": true,
+				"KeepValues": [
+					"a",
+					"b",
+					"c"
+				],
+				"ObfuscateSQLValues": [
+					"x",
+					"y"
+				]
+			},
+			"Mongo": {
+				"Enabled": true,
+				"KeepValues": [
+					"a",
+					"b",
+					"c"
+				],
+				"ObfuscateSQLValues": [
+					"x",
+					"y"
+				]
+			},
+			"SQLExecPlan": {
+				"Enabled": true,
+				"KeepValues": [
+					"a",
+					"b",
+					"c"
+				],
+				"ObfuscateSQLValues": [
+					"x",
+					"y"
+				]
+			},
+			"SQLExecPlanNormalize": {
+				"Enabled": true,
+				"KeepValues": [
```

Comment:
I think we'll use the detailed SQL obfuscation info.
I think we mostly just need off or on, but I still curious how to interpret these values.

Reply:
I will provide references to what all of these mean once we agree on a final form for this PR.